### PR TITLE
Update and Query commands are added to the files as per requirements

### PR DESCRIPTION
**Pull request by Metaverse-Nabeel**
### The following changes have been made to this PR:

- New entries are added to the tables
image

- First Group of Screenshots for Transaction
image
image

- Second Group of Screenshots for Transaction
image

- Third Group of Screenshots for Transaction
image
image

- Fourth Group of Screenshots for Queries
image
image

Thank you :)

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -43,4 +43,93 @@ SELECT *
...
+
+ROLLBACK TRANSACTION;
+
+SELECT * FROM animals;
+
+-- Transaction for first screenshot
+
+BEGIN TRANSACTION;
+
+UPDATE animals SET species = 'digimon' WHERE "name" LIKE '%mon';
```
Comment (Zuhaib042) [inline queries.sql:64]: - [x] There is some issue here, as per this query and requirement, all the animals that have `mon` at the end of their names should have a species of digimon. but in your case two animals `Agumon and Devimon` have species of Pokemon which is wrong. they should have species of `digimon`. kindly consider correcting that.
qw

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @Metaverse-Nabeel 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Descriptive PR
✅ Correct Gitflow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Metaverse-Nabeel) [reply queries.sql:64]: Hello @Zuhaib042 

I hope you are doing well. Thank you so much for highlighting this issue. I already knew about this issue and have tried many different ways to solve but could not understand why it is not working properly. Could you please guide me on how to correct it? I have already searched and I have the same command for this query from every source.

Anxiously waiting for your response.

Comment (Metaverse-Nabeel) [reply queries.sql:64]: Hello @Zuhaib042 

I have made the correction. There was an issue in inserting the data. At the end of the animal names, there was a blank space.

Now, the command is working perfectly fine. Here is the screenshot.
image

Thank you

**Review by Zuhaib042 — APPROVED** [review]
Hi @Metaverse-Nabeel 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Successfully fixed the bug reported by the reviewer
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**